recipe/meta.yaml

```diff
@@ -23,11 +23,13 @@ build:
     - vc9    # [win and py27]
     - vc10   # [win and py34]
     - vc14   # [win and py35]
+    - readline 6.3
 
 requirements:
   build:
     # python is for lining up vc feature (runtime)
     - python      # [win]
+    - readline 6.3
```

Comment:
Please change to `6.2` to match our [global pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/70036d169783f7d6844bfc1ff1b360dba5adfa9f/scripts/pin_the_slow_way.py#L48 ).